flow/src/test/java/com/sequenceiq/flow/EnforceEntityDenialUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.sequenceiq.flow;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.persistence.Entity;
+
+import org.reflections.Reflections;
+import org.reflections.scanners.SubTypesScanner;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Sets;
+import com.sequenceiq.cloudbreak.common.event.IdempotentEvent;
+import com.sequenceiq.cloudbreak.common.event.Payload;
+
+public class EnforceEntityDenialUtil {
+
+    private static final Reflections REFLECTIONS = new Reflections("com.sequenceiq",
+            new SubTypesScanner(false));
+
+    private EnforceEntityDenialUtil() {
+
+    }
+
+    public static void denyEntity() {
+        Set<Field> entityFieldsInPayload = Sets.union(REFLECTIONS.getSubTypesOf(Payload.class), REFLECTIONS.getSubTypesOf(IdempotentEvent.class))
+                .stream()
+                .map(Class::getDeclaredFields)
+                .flatMap(fields -> Arrays.stream(fields))
+                .filter(field -> field.getType().isAnnotationPresent(Entity.class))
+                .collect(Collectors.toSet());
+        assertTrue(entityFieldsInPayload.isEmpty(), Joiner.on("\n").join(entityFieldsInPayload));
```

Comment:
Some explanation might be nice to have in the assert like:
"You have tried to include an Entity object in a FlowRequest which would cause runtime issues."

I'm fine with adding this later, just a hint.

Reply:
yes, right, I wanted to add it, but forgot